### ISSUE TITLE
Incompatible with stylelint version 14

### ISSUE THREAD

**Issue by rrelmy** [tyler36/laravel-mix-stylelint#5]
stylelint [version 14](https://github.com/stylelint/stylelint/blob/14.0.0/docs/migration-guide/to-14.md) removed the syntax option.

The option is used here
https://github.com/tyler36/laravel-mix-stylelint/blob/c389cbafc471b1d6d65d762c8c4fd9dc60c4f504/index.js#L47

Using the default settings throws following error during builds
```
ERROR in The "syntax" option is no longer available. You should install an appropriate syntax, e.g. postcss-scss, and use the "customSyntax" option
```

## Workaround
Passing a explictly empty value overrides the default and makes stylelint happy.

``` js
mix.stylelint({ syntax: null });
```

**Comment by tyler36** [comment]
Thank you for the update and workaround.

Hopefully, I'll find some time to check the migration path and add update the package. Feel free to submit a PR if you feel like contributing.

I mostly use TailwindCSS these days so I don't do a lot of CSS linting.

**Comment by tyler36** [comment]
Fixed in #6
